components/match2/wikis/pokemon/brkts_wikispecific.lua

```diff
@@ -0,0 +1,30 @@
+---
+-- @Liquipedia
+-- wiki=pokemon
+-- page=Module:Brkts/WikiSpecific
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+
+local _EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
+
+local WikiSpecific = Table.copy(Lua.import('Module:Brkts/WikiSpecific/Base', {requireDevIfEnabled = true}))
+
+--
+-- Override functons
+--
+function WikiSpecific.matchHasDetails(match)
+	return match.dateIsExact
+		or match.date ~= _EPOCH_TIME_EXTENDED
+		or match.vod
+		or not Table.isEmpty(match.links)
+		or match.comment
+		or 0 < #match.games
+end
+
+WikiSpecific.defaultIcon = 'Pokemon allmode.png'
```

Comment:
Where's this used?

Reply:
Im unsure too as other moba has these four specific lines as well, removing it on pokemon doesnt seem to change anything as well. The match.games at least has its mentions in both Legacy and Summary modules

Reply:
It's just the last one I'm asking about :)
Github always shows 4 lines to get some context

Reply:
Try to remove it and see if anything breaks for TBD's. If not you can remove it.